libs/bluegl/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ project(bluegl ASM)
 set(TARGET              bluegl)
 set(PUBLIC_HDR_DIR      include)
 
-if (WIN32)
+if (WIN32 AND CMAKE_SIZEOF_VOID_P EQUAL 8)
```

Comment:
Instead of the cryptic `CMAKE_SIZEOF_VOID_P EQUAL 8` is would be better to create an intermediate variable called IS_64_BIT.

Reply:
Thanks. I will update.